src/mod_muc_room.erl

```diff
@@ -4071,6 +4073,35 @@ is_privacy_allow(Packet) ->
     LServer = To#jid.server,
     allow == ejabberd_hooks:run_fold(privacy_check_packet, LServer, allow, [To, Packet, in]).
 
+-spec inspect_sdk_xmlels(binary(), map(), list()) -> list().
```

Comment:
Can we add more doco about specs?

Can you add a pull request to community edition too? so that you can get feedback from the community? especially we do not have in house experts on this, yet...

https://github.com/processone/ejabberd/blob/master/src/mod_muc_room.erl

Reply:
Talked offline, will make a forum post